uncontrast_studies/processors/experiments_comparison.py

```diff
@@ -0,0 +1,225 @@
+        subquery = (
+            experiments.distinct()
+            .values("paradigm")
+            .annotate(experiment_count=Count("id", distinct=True))
+            .annotate(key=F("paradigm"))
+        )
+        return subquery
+
+    def process_population(self, experiments: QuerySet[UnConExperiment]):
+        subquery = (
+            UnConSample.objects.filter(experiment__in=experiments)
+            .distinct()
+            .values("type")
+            .annotate(experiment_count=Count("experiment", distinct=True))
+            .annotate(key=F("type"))
+        )
+
+        return subquery
+
+    def process_task(self, experiments: QuerySet[UnConExperiment]):
+        subquery = (
+            UnConTaskType.objects.filter(tasks__experiment__in=experiments)
+            .distinct()
+            .values("name")
+            .annotate(experiment_count=Count("tasks__experiment", distinct=True))
+            .annotate(key=F("name"))
+        )
+
+        return subquery
+
+    def process_suppressed_stimuli_category(self, experiments: QuerySet[UnConExperiment]):
+        subquery = (
+            UnConStimulusCategory.objects.filter(suppressed_stimuli__experiment__in=experiments)
+            .distinct()
+            .values("name")
+            .annotate(experiment_count=Count("suppressed_stimuli__experiment", distinct=True))
+            .annotate(key=F("name"))
+        )
+
+        return subquery
+
+    def process_target_stimuli_category(self, experiments: QuerySet[UnConExperiment]):
+        subquery = (
+            UnConStimulusCategory.objects.filter(target_stimuli__experiment__in=experiments)
+            .distinct()
+            .values("name")
+            .annotate(experiment_count=Count("target_stimuli__experiment", distinct=True))
+            .annotate(key=F("name"))
+        )
+
+        return subquery
+
+    def process_suppressed_stimuli_modality(self, experiments: QuerySet[UnConExperiment]):
+        subquery = (
+            UnConModalityType.objects.filter(suppressed_stimuli__experiment__in=experiments)
+            .distinct()
+            .values("name")
+            .annotate(experiment_count=Count("suppressed_stimuli__experiment", distinct=True))
+            .annotate(key=F("name"))
+        )
+
+        return subquery
+
+    def process_target_stimuli_modality(self, experiments: QuerySet[UnConExperiment]):
+        subquery = (
+            UnConModalityType.objects.filter(target_stimuli__experiment__in=experiments)
+            .distinct()
+            .values("name")
+            .annotate(experiment_count=Count("target_stimuli__experiment", distinct=True))
+            .annotate(key=F("name"))
+        )
+
+        return subquery
+
+    def process_consciousness_measure_phase(self, experiments: QuerySet[UnConExperiment]):
+        subquery = (
+            UnConsciousnessMeasurePhase.objects.filter(unconsciousness_measures__experiment__in=experiments)
```

Comment:
בטוח שזה unconsciousness_measures ולא unconsciousness_measure? וגם בכל המקומות בהמשך

Reply:
Yes, it's a bit strange, but it seems so, as it's the "reverse relation"